builtin/providers/aws/resource_aws_elb.go

```diff
@@ -305,6 +328,11 @@ func resourceAwsElbRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("idle_timeout", lbAttrs.ConnectionSettings.IdleTimeout)
 	d.Set("connection_draining", lbAttrs.ConnectionDraining.Enabled)
 	d.Set("connection_draining_timeout", lbAttrs.ConnectionDraining.Timeout)
+	if lbAttrs.AccessLog != nil {
+		if err := d.Set("access_logs", flattenAccessLog(lbAttrs.AccessLog)); err != nil {
+			log.Printf("[WARN] Error setting ELB Access Logs for (%s): %s", d.Id(), err)
```

Comment:
If we end up with an error here - we probably want to return it, no? Seems like an error here would be a bug we'd need to fix, so better to make it obvious.